test/test_spherical_advection.jl

```diff
@@ -7,21 +7,49 @@ include("test_trixiatmo.jl")
 
 EXAMPLES_DIR = pkgdir(TrixiAtmo, "examples")
 
-@trixiatmo_testset "Spherical advection, Cartesian weak form, LLF surface flux" begin
+@trixiatmo_testset "Spherical advection (cubed sphere), Cartesian weak form, LLF surface flux" begin
     @test_trixi_include(joinpath(EXAMPLES_DIR,
                                  "elixir_shallowwater_cubed_sphere_shell_advection.jl"),
                         l2=[
-                            0.7963221028128544,
-                            20.317489255709344,
-                            8.811382597221147,
-                            20.317512894705885,
+                            0.796321633853675,
```

Comment:
Is this due to the `a = sqrt(x[1]^2 + x[2]^2 + x[3]^2)` in `initial_condition_gaussian`?

Reply:
Yes. Even though this test case did not use a `refinement_level > 0`, the Cartesian analysis routines evaluate the error on a different set of nodes with `analysis_polydeg = 2 * polydeg(basis)` by default. The analysis node positions are obtained with polynomial interpolation inside Trixi.